ONSdigital/dp-kafka
# Added v2 to requires and imports, to comply with golang semver

### What

- Added v2 to requires and imports, to comply with golang semver
  - More information: https://github.com/golang/go/wiki/Modules#semantic-import-versioning

### How to review

- Make sure `v2` is added to the requires and imports
- Make sure go.mod and go.sum don't have dependencies with dp-kafka without v2 suffix.

### Who can review

Anyone
